### PR TITLE
api: add instructions to run examples

### PR DESCRIPTION
**Pull request by jirikuncar**
### What does this PR do?

Adds instructions how to run API code examples.

### Motivation

- `DD_SITE` needs to be exported
- command and file name might not be obvious

### Preview

https://docs-staging.datadoghq.com/jirikuncar/dd-site/api/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: layouts/partials/api/code-example.html [hunk 1]
```diff
@@ -72,12 +72,20 @@ <h3 class="mb-2">{{ i18n "code_example" }}</h3>
...
+            <div class="code-snippet">
               <div class="code-button-wrapper position-absolute">
                   <button class="btn text-primary js-copy-button">{{ i18n "copy" }}</button>
               </div>
               {{ highlight $file_content $lang.syntax "" }}
-          </div>
+            </div>
+
+            <h4>Instructions</h4>
+            <p>Save example to <code>{{ $lang.example_file }}</code> and run following commands.</p>
```
Comment (kayayarai) [inline layouts/partials/api/code-example.html:83]:
```suggestion
            <p>Save the example to <code>{{ $lang.example_file }}</code> and run following commands:</p>
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Minor edit

**Review by davidejones — APPROVED** [review]
nice! looks good here

**Review by kayayarai — APPROVED** [review]
🎉 Looks great!

**Pull request merged**